src/components/aeAddressInput/aeAddressInput.scss

```diff
@@ -1,6 +1,15 @@
 @import '../variables.scss';
-label {
+@import '../_extendables.scss';
+
+input{
+  @extend %input;
```

Comment:
`aeAddressInput.scss` is imported to `aeAddressInput.vue` without `scoped` atribute and this changes behavour of all inputs in project.